CMakeLists.txt

```diff
@@ -1,7 +1,8 @@
 cmake_minimum_required(VERSION 3.0)
 
+
 if(NOT CMAKE_BUILD_TYPE)
-	set(CMAKE_BUILD_TYPE "Release" CACHE STRING
+	set(CMAKE_BUILD_TYPE "Debug" CACHE STRING
```

Comment:
Please set default build to Release as most users expect a Release build as default.